cs3api4lab/auth/authenticator.py

```diff
@@ -0,0 +1,32 @@
+import cs3.gateway.v1beta1.gateway_api_pb2 as cs3gw
+import time
+import cs3.gateway.v1beta1.gateway_api_pb2_grpc as cs3gw_grpc
+from cs3api4lab.config.config_manager import Cs3ConfigManager
+from cs3api4lab.auth.channel_connector import ChannelConnector
+
+
+class Authenticator:
+    tokens = {}  # map client_id [string] to {authentication token, token expiration time}
+
+    def __init__(self):
+        self.config = Cs3ConfigManager().config
```

Comment:
Why are you reading the config again? Ans setting another channel? This could be passed by parameter, from the component that has already read it/has it.

I understand we are trying to make these things independent, but I think this shouldn't be.
We should't keep reading the same thing again...